uyuni-docs-helper

```diff
@@ -120,37 +120,37 @@ while true ; do
 done
 
 # Command is mandatory
-if [ -z ${COMMAND} ]; then
+if [ -z "${COMMAND}" ]; then
   print_incorrect_syntax
 fi
 
 # Print a special error for the help command
 if [ "${COMMAND}" == "help" ]; then
-  if [ -z ${LOCALCLONE} ] && [ -z ${GITREPO} ]; then
+  if [ -z "${LOCALCLONE}" ] && [ -z "${GITREPO}" ]; then
     print_error "A source (a remote or local git repository) is needed for this."
     print_incorrect_syntax
   # Define a default product, if none is available, so we can show the help
-  elif [ -z ${PRODUCT} ]; then
+  elif [ -z "${PRODUCT}" ]; then
     PRODUCT='suma'
   fi
 fi
 
 # Product is mandatory
-if [ -z ${PRODUCT} ]; then
+if [ -z "${PRODUCT}" ]; then
   print_incorrect_syntax
 fi
 
 # Either a remote source or a local one must be used
-if [ -z ${LOCALCLONE} ]; then
-  if [ -z ${GITREPO} ]; then
+if [ -z "${LOCALCLONE}" ]; then
+  if [ -z "${GITREPO}" ]; then
     print_incorrect_syntax
   fi
-  if [ -z ${GITREF} ]; then
+  if [ -z "${GITREF}" ]; then
     print_incorrect_syntax
   fi
   SOURCE="-e GITREPO=${GITREPO} -e GITREF=${GITREF}"
-  if [ ! -z ${OUTPUT} ]; then
-    if [ ! -d ${OUTPUT} ]; then
+  if [ -n "${OUTPUT}" ]; then
```

Comment:
Conceptually speaking `! -z` and `-n`, as one checks if a variable exists, while the other checks if the variable is empty.

But should work anyway, so OK.

Reply:
Correct. The [Shellcheck reference](https://www.shellcheck.net/wiki/SC2236) states:
> This is a stylistic issue that does not affect correctness. If you prefer the original expression, you can [Ignore](https://www.shellcheck.net/wiki/Ignore) it with a directive or flag.